shared/cloud/ess-access-kibana.asciidoc

```diff
@@ -0,0 +1,26 @@
+// Include this file in your docs:
+// include::{docs-root}/shared/cloud/ess-access-kibana.asciidoc[]
+//
+// Use a different heading level, e.g. H3? 
+// :leveloffset: +1 
+// include::{docs-root}/shared/cloud/ess-getting-started.asciidoc[]
+// :leveloffset: 0
```

Comment:
Now that headings aren't being included in these files, is this necessary?

Reply:
Yes, good point. Removed these :leveloffset: instructions here and in the other single-sourced file.